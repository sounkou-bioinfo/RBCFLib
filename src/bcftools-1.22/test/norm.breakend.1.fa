>chrA
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
ACTGACTGACTGACTGACTGCCCCCACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGA
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
ACTGACTGACTGACTGACTGTTTTTACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGA
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN

>chrB
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
ACTGACTGACTGACTGACTGCCCCCACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGA
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
ACTGACTGACTGACTGACTGTTTTTACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGAACTGA
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
