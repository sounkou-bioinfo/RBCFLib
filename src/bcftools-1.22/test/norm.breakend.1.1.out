##fileformat=VCFv4.5
##FILTER=<ID=PASS,Description="All filters passed">
##INFO=<ID=MATEID,Number=A,Type=String,Description="ID of mate breakend">
##contig=<ID=chrA,length=500>
##contig=<ID=chrB,length=500>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chrA	321	bnd_V	T	]chrB:123]AGTNNNNNCAT	6	PASS	MATEID=bnd_U
chrB	123	bnd_U	C	CAGTNNNNNCA[chrA:321[	6	PASS	MATEID=bnd_V
